apps/OpenSpace/ext/launcher/src/profile/additionalscriptsdialog.cpp

```diff
@@ -26,6 +26,7 @@
 
 #include "profile/line.h"
 #include "profile/scriptlogdialog.h"
+#include <ghoul/misc/stringhelper.h>
 #include <openspace/scene/profile.h>
```

Comment:
```suggestion
#include <openspace/scene/profile.h>
#include <ghoul/misc/stringhelper.h>
```

Include file sorting